public/styles.css

```diff
@@ -224,7 +224,7 @@ a:hover {
 
 /* School and bio cards */
 .card {
-  width: 1170px;
+  max-width: 100%;
```

Comment:
is this necessary? what is the max-width otherwise? what happens if u delete this